### DISCUSSION TITLE
Is it possible to use background task in WinUI3 unpackage desktop app?

### DISCUSSION THREAD

**Discussion by gscheartA** [microsoft/microsoft-ui-xaml#7624] [Q&A]
I am developing a unpackage desktop app about WinUI3. I want to use some feature , such as network background task or daily task in uwp. Is it possbile to use it in unpackage desktop app? Thank you for your answer.

**Comment by anovik** [comment]
Is there any update regarding this? What is the best way currently to support background tasks (similar to UWP) in WinUI3?
